kura/test/org.eclipse.kura.rest.packages.provider.test/pom.xml

```diff
@@ -408,31 +408,6 @@
                             <level>4</level>
                             <autoStart>true</autoStart>
                         </bundle>
-                        <bundle>
```

Comment:
I think that you can remove all the configuration of these plugins, since already defined in parent. Or do you need a specific bundle to run the tests?

Something like this should suffice:
```xml
    <build>
        <plugins>
            <plugin>
                <groupId>org.jacoco</groupId>
                <artifactId>jacoco-maven-plugin</artifactId>
            </plugin>
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-compiler-plugin</artifactId>
                <executions>
                    <execution>
                        <id>compiletests</id>
                        <phase>test-compile</phase>
                        <goals>
                            <goal>testCompile</goal>
                        </goals>
                    </execution>
                </executions>
            </plugin>
            <plugin>
                <groupId>org.eclipse.tycho</groupId>
                <artifactId>tycho-surefire-plugin</artifactId>
            </plugin>
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-surefire-plugin</artifactId>
            </plugin>
            <plugin>
                <groupId>org.eclipse.tycho</groupId>
                <artifactId>target-platform-configuration</artifactId>
            </plugin>
        </plugins>
    </build>
```

Reply:
Hi @mattdibi @marcellorinaldo  I tried this option, and I can confirm that worked for me. I perform the build on a Docker container (maven:3.8.5-openjdk-17) with the tests as well, and experienced no issues this time.

Reply:
@marcellorinaldo as per our discussion we'll keep it as-is for now.